www-client/qutebrowser/qutebrowser-9999.ebuild

```diff
@@ -24,6 +24,7 @@ IUSE="test"
 COMMON_DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
 DEPEND="${COMMON_DEPEND}
 	app-text/asciidoc
+	dev-util/source-highlight
```

Comment:
This shouldn't be necessary anymore, see     https://github.com/qutebrowser/qutebrowser/commit/dc01b4eaf0e945b131e8f8094d1a7b6a5c873c1c